### ISSUE TITLE
conf/common.py: Incorrect IP address returned by getIpByHostInSubnet()

### ISSUE THREAD

**Issue by ommoreno** [01org/CeTune#173]
CeTune uses getIpByHostInSubnet() method in conf/common.py to grab the IP address of the specified host. If the host has a non-localhost interface(s) (i.e. docker/VM bridge), then that will be used even if it is not on the correct network defined in ceph.conf.

**Comment by xuechendi** [comment]
Yes, and I believe which is already fixed by using ifconfig in master, could you check on it? Thanks

在 2017年1月26日，05:19，Orlando Moreno <notifications@github.com<mailto:notifications@github.com>> 写道：


CeTune uses getIpByHostInSubnet() method in conf/common.py to grab the IP address of the specified host. If the host has a non-localhost interface(s) (i.e. docker/VM bridge), then that will be used even if it is not on the correct network defined in ceph.conf.

―
